src/test/java/org/opensearch/neuralsearch/search/query/HybridQueryPhaseSearcherTests.java

```diff
@@ -1025,7 +1025,9 @@ private static IndexMetadata getIndexMetadata() {
             RemoteStoreEnums.PathType.NAME,
             HASHED_PREFIX.name(),
             RemoteStoreEnums.PathHashAlgorithm.NAME,
-            RemoteStoreEnums.PathHashAlgorithm.FNV_1A_BASE64.name()
+            RemoteStoreEnums.PathHashAlgorithm.FNV_1A_BASE64.name(),
+            IndexMetadata.TRANSLOG_METADATA_KEY,
```

Comment:
Added this setting to cater https://github.com/opensearch-project/OpenSearch/commit/76e2a5d9d70ab310033f33151fdaa0ab68c6537e#diff-240fea2643bc047fd12adfba28f9ba4e002bd0935e62b4f6fe49465526c3dc07R190